### PR TITLE
Awaymissions are now a memory of the past (almost)

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

This PR just removes the Awaymission maps and content, since they're unused.

## Why It's Good For The Game

Faster load time, Less headaches for pathchanges and mappers.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

I.O.U. a screenshot, but it all works 👌 

</details>

## Changelog
:cl:
del: Removed the awaymission content and maps
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
May I make a special request to specifically keep some of the mobs that we use as admins, or should I just re-add them in a separate PR?

**Comment by MarkusLarsson421** [comment]
If I am not mistaken, a lot of those mobs are awfully organized. So I think they should just be readded.

**Comment by PestoVerde322** [comment]
> May I make a special request to specifically keep some of the mobs that we use as admins, or should I just re-add them in a separate PR?

Sure which mobs i should reimplement?

**Comment by Rukofamicom** [comment]
The heavy gunner and sniper. 
I believe they both came from the factory but I'm not certain (there is a real possibility they aren't removed here in the first place, but I'm pretty certain they came from away missions)

I have not done a look at code yet, if you want to wait for me to check and see if they were even included in this give me a few days to do a proper review and I'll point them out when I do.

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -542,3 +542,74 @@
 		/obj/item/stack/ore/uranium = 2,
 		/obj/item/stack/ore/diamond = 2)
 
+///Former Snowdin loot tables
+
+/obj/effect/spawner/lootdrop/snowdin
```
Comment (PowerfulBacon) [inline code/game/objects/effects/spawners/lootdrop.dm:547]: Do we use these?

Comment (PestoVerde322) [reply code/game/objects/effects/spawners/lootdrop.dm:547]: yep, in some random ruins in explo!

File: code/modules/awaymissions/mission_code/TheFactory.dm [hunk 2]
```diff
@@ -1,835 +0,0 @@
...
-	loot = list(/obj/item/gun/ballistic/automatic/wt550,
-				/obj/effect/mob_spawn/human/corpse/nanotrasensoldier)
-
-/mob/living/simple_animal/hostile/syndicate/factory
-	gender = MALE
-	faction = list("nanotrasenprivate")
-	speak_chance = 5
-	speak = list("Come get me!","You can't get away!","Die!")
-
-/mob/living/simple_animal/hostile/syndicate/factory/sniper
```
Comment (Rukofamicom) [inline code/modules/awaymissions/mission_code/TheFactory.dm:339]: This is the sniper

File: code/modules/awaymissions/mission_code/TheFactory.dm [hunk 3]
```diff
@@ -1,835 +0,0 @@
...
-/mob/living/simple_animal/hostile/psycho/trap/death(gibbed)
-	var/list/possible_sounds = list('sound/creatures/psychdeath.ogg','sound/creatures/psychdeath2.ogg')
-	var/chosen_sound = pick(possible_sounds)
-	playsound(get_turf(src), chosen_sound, 70, 0, 0)
-	playsound(get_turf(src), 'sound/effects/snap.ogg', 75, TRUE, 0)
-	playsound(get_turf(src), 'sound/effects/splat.ogg', 90, TRUE, 0)
-	visible_message("<span class='boldwarning'>The device activates!</span>")
-	..()
-
-/mob/living/simple_animal/hostile/syndicate/factory/heavy
```
Comment (Rukofamicom) [inline code/modules/awaymissions/mission_code/TheFactory.dm:548]: This is the heavy gunner.

File: code/modules/mob/living/simple_animal/corpse.dm [hunk 4]
```diff
@@ -272,58 +250,7 @@
...
-	outfit = /datum/outfit/minigunheavy
-
-/datum/outfit/minigunheavy
-	name = "Heavy gunner"
-	uniform = /obj/item/clothing/under/rank/security/head_of_security/alt
-	shoes = /obj/item/clothing/shoes/combat/swat
-	gloves = /obj/item/clothing/gloves/combat
-	mask = /obj/item/clothing/mask/gas/sechailer/swat/emagged
-	suit = /obj/item/clothing/suit/armor/heavy
-	back = /obj/item/minigunpack
```
Comment (Rukofamicom) [inline code/modules/mob/living/simple_animal/corpse.dm:291]: And this is the minigun backpack the heavy gunner should not spawn with, and should still be removed. 

It's broken shitcode I fixed up once, but it really doesn't need to exist for players to use.

**Review by Rukofamicom — COMMENTED** [review]
Pointing out the two mobs I would like to cherry pick for admin spawning reasons.

**Review by Rukofamicom — APPROVED** [review]
Even if you opt not to keep those, we can always bring them back later ourselves if we really want to. 

When conflicts are fixed up this is a good cleanup.

**Comment by Rukofamicom** [comment]
Just have to fix up the conflicts and this is good to merge.
We can always re-add them later

**Comment by Rukofamicom** [comment]
What changes does this make to station maps exactly?

**Review by spockye — APPROVED** [review]
checked off map-wise

**Comment by Rukofamicom** [comment]
fix conflict and ping

**Pull request merged**